content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -37,73 +37,89 @@ further_reading:
+  <p style="font-weight: bold; margin-bottom: 0px;">Note:<p/> The .NET runtime tries to load a profiler into any .NET process started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. 
+  <p style="font-weight: bold; margin-bottom: 0px">We do not recommend setting these environment variables globally as this causes all .NET processes on the host to load the profiler.<p/>
+</div>
 
-For Windows applications **not** running in IIS, set these two environment variables before starting your application to enable automatic instrumentation:
+#### Non-IIS Applications
+
+For Windows applications, set these two environment variables before starting your application to enable automatic instrumentation:
 
-**Note:** The .NET runtime tries to load a profiler into _any_ .NET process that is started with these environment variables set. You should limit instrumentation only to the applications that need to be traced. **We do not recommend setting these environment variables globally as this causes _all_ .NET processes on the host to load the profiler.**
 
 | Name                   | Value                                    |
 | ---------------------- | ---------------------------------------- |
 | `COR_ENABLE_PROFILING` | `1`                                      |
 | `COR_PROFILER`         | `{846F5F1C-F9AE-4B07-969E-05C26BC060D8}` |
 
+
 #### Windows Services
+To automatically instrument a Windows Service:
 
-To automatically instrument a Windows Service, set the environment variables for that Service in the Windows Registry. Create a multi-string value called `Environment` in the `HKLM\System\CurrentControlSet\Services\<SERVICE NAME>` key. Then set the key's data to the values in the table:
-```text
-COR_ENABLE_PROFILING=1
-COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
-```
+1. Set the environment variables for the Service in the Windows Registry.
+2. Create a multi-sting value called `Environment` in the `HKLM\System\CurrentControlSet\Services\<SERVICE NAME>`
+3. Set the key's data to the value of: 
 
-This can be done either through the Registry Editor as in the image below, or through a PowerShell snippet:
+   ```text
+   COR_ENABLE_PROFILING=1
+   COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+   ```
 
-{{< img src="tracing/setup/dotnet/RegistryEditorFramework.png" alt="Registry Editor"  >}}
+Setting the `COR_ENABLE_PROFILING` & `COR_PROFILER` environment variables can be achieved by:
 
-{{< code-block lang="powershell" filename="add-env-var.ps1" >}}
-[String[]] $v = @("COR_ENABLE_PROFILING=1", "COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}")
-Set-ItemProperty HKLM:SYSTEM\CurrentControlSet\Services\<NAME> -Name Environment -Value $v
-{{< /code-block >}}
+1. Through the Windows Registry Editor:
 
-#### Console Apps
+     {{< img src="tracing/setup/dotnet/RegistryEditorFramework.png" alt="Registry Editor"  >}}
```

Comment:
In addition to the screen capture, we should also put this in words, so that someone using a screen reader doesn't miss out on the information.